lib/passfort.rb

```diff
@@ -8,6 +8,21 @@
 require "passfort/client"
 
 module Passfort
+  module CheckType
+    IDENTITY_CHECK = "IDENTITY_CHECK"
+    DOCUMENT_VERIFICATION = "DOCUMENT_VERIFICATION"
+    DOCUMENT_FETCH = "DOCUMENT_FETCH"
+    PEPS_SCREEN = "PEPS_SCREEN"
+    SANCTIONS_SCREEN = "SANCTIONS_SCREEN"
```

Comment:
I couldn't find a reference to `PEPS_SCREEN` or `SANCTIONS_SCREEN` in the docs - are you sure they exist?

Reply:
https://passfort.com/developer/v4/data-reference/CheckResourcePepsScreen
https://passfort.com/developer/v4/data-reference/CheckResourceSanctionsScreen

Reply:
Those are `CheckResources`, not check types - can you find a corresponding check type for those? Because I can't.

Reply:
API Reference: https://passfort.com/developer/v4/data-reference/CheckType
Though, as tested, not all seem to work.